diff_drive_controller/include/diff_drive_controller/diff_drive_controller.hpp

```diff
@@ -88,10 +88,11 @@ class DiffDriveController : public controller_interface::ControllerInterface
 protected:
   struct WheelHandle
   {
-    std::reference_wrapper<const hardware_interface::LoanedStateInterface> position;
+    std::reference_wrapper<const hardware_interface::LoanedStateInterface> feedback;
     std::reference_wrapper<hardware_interface::LoanedCommandInterface> velocity;
```

Comment:
We should probably also rename this in the follow-up PR to make the variable name clearer.

Reply:
You mean choosing a different name than "feedback"? Suggestions?

Reply:
I think it's ok for now

Reply:
I am actually proposing to rename the "velocity" to "command" or something like this.